src/pq_flash_index.cpp

```diff
@@ -1766,6 +1766,20 @@ template <typename T, typename LabelT> char *PQFlashIndex<T, LabelT>::getHeaderB
 }
 #endif
 
+template <typename T, typename LabelT>
+size_t PQFlashIndex<T, LabelT>::search_string_range(const std::string &str, char ch, size_t start, size_t end)
+{
+    for (; start != end; start++)
+    {
+        if (str[start] == ch)
+        {
+            return start;
+        }
+    }
+
+    return std::string::npos;
+}
+
```

Comment:
![image](https://github.com/microsoft/DiskANN/assets/15103725/c8ddb24f-2afb-4ef9-be3a-dc083535fff3)

Reply:
so strange to have this error

Reply:
yes, not sure of this. will check later